### PR TITLE
chore: move sign_document.js file from bloomstack_core to erpnext

### PR DESCRIPTION
**Pull request by meshramsaroj**
**File Name:** bloomstack_core/bloomstack_core/templates/includes/sign_document.js

**Deleted file PR:** https://github.com/Bloomstack/bloomstack_core/pull/697

**Dependent PR:** https://github.com/Bloomstack/erpnext/pull/1632 https://github.com/Bloomstack/erpnext/pull/1659 https://github.com/Bloomstack/erpnext/pull/1643

**Dependent files method:**
erpnext/utils
- authorize_document

erpnext/www/authorize_document
- custom_print_doc

### CODE REVIEW

**Comment by sahil28297** [comment]
@meshramsaroj add proper PR title

File: erpnext/templates/includes/sign_document.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		}
+
+		// proceed only if user has put name, signature and designation
+		if (!(signee && $sigdiv.jSignature("getData", "native").length > 0 && designation)) {
+			frappe.throw(__("Please enter your name, signature and designation"));
+		}
+
+		$(".user-signature").hide();
+		frappe.call({
+			method: "erpnext.utils.authorize_document",
```
Comment (sahil28297) [inline erpnext/templates/includes/sign_document.js]: The path is not proper, co-ordinate with Vishal for the correct path

File: erpnext/templates/includes/sign_document.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+				sign: sign,
+				signee: signee,
+				docname: "{{ auth_req_docname }}",
+				party_business_type: party_business_type,
+				designation: designation
+			},
+			freeze: true,
+			callback: (r) => {
+				frappe.call({
+					method: "erpnext.www.authorize_document.custom_print_doc",
```
Comment (sahil28297) [inline erpnext/templates/includes/sign_document.js:57]: Co-ordinate with Mrunmay for the correct path.

File: erpnext/templates/includes/sign_document.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+					}
+				});
+			}
+		});
+	});
+
+	$("#rejectDocument").on("click", function () {
+		$(".user-signature").hide();
+		frappe.call({
+			method: "erpnext.utils.reject_document",
```
Comment (sahil28297) [inline erpnext/templates/includes/sign_document.js]: The path is not proper, co-ordinate with Vishal for the correct path

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/templates/includes/sign_document.js (hunk 1 above)
Comment (meshramsaroj) [reply erpnext/templates/includes/sign_document.js]: file path is fixed as per dependant PR

File: erpnext/templates/includes/sign_document.js (hunk 2 above)
Comment (meshramsaroj) [reply erpnext/templates/includes/sign_document.js:57]: file path is fixed as per dependant PR https://github.com/Bloomstack/erpnext/pull/1659

File: erpnext/templates/includes/sign_document.js (hunk 3 above)
Comment (meshramsaroj) [reply erpnext/templates/includes/sign_document.js]: file path is fixed as per dependant PR https://github.com/Bloomstack/erpnext/pull/1643

**Review by sahil28297 — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**